astropy/io/fits/convenience.py

```diff
@@ -408,9 +410,11 @@ def writeto(filename, data, header=None, output_verify='exception',
         ``+warn``, or ``+exception" (e.g. ``"fix+warn"``).  See :ref:`verify`
         for more info.
 
-    clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+    overwrite : bool, optional
+        If ``True``, overwrite the output file if exists.
```

Comment:
I like the suggestion about "raises an OSError (IOError on Python 2) if the file exists and overwrite is False." but I don't like the ``Raises`` section if it's completly dependant on the _one_ parameter.